base/docs/helpdb/Base.jl

```diff
@@ -3320,23 +3320,38 @@ it will launch as many workers as the number of cores on the specific host.
 
 Keyword arguments:
 
-`tunnel`: if `true` then SSH tunneling will be used to connect to the worker from the
-master process.
+* `tunnel`: if `true` then SSH tunneling will be used to connect to the worker from the
+            master process. Default is `false`.
 
-`sshflags`: specifies additional ssh options, e.g.
+* `sshflags`: specifies additional ssh options, e.g.
 
     sshflags=`-i /home/foo/bar.pem`
 
-`max_parallel`: specifies the maximum number of workers connected to in parallel at a host.
-Defaults to 10.
+* `max_parallel`: specifies the maximum number of workers connected to in parallel at a host.
+                  Defaults to 10.
 
-`dir`: specifies the working directory on the workers. Defaults to the host's current
-directory (as found by `pwd()`)
+* `dir`: specifies the working directory on the workers. Defaults to the host's current
+         directory (as found by `pwd()`)
 
-`exename`: name of the julia executable. Defaults to `"\$JULIA_HOME/julia"` or
-`"\$JULIA_HOME/julia-debug"` as the case may be.
+* `exename`: name of the julia executable. Defaults to `"\$JULIA_HOME/julia"` or
+             `"\$JULIA_HOME/julia-debug"` as the case may be.
+
+* `exeflags`: additional flags passed to the worker processes.
+
+* `topology`: Specifies how the workers connect to each other. Sending a message
+            between unconnected workers results in an error.
+
+  + `:all_to_all`  :  All processes are connected to each other.
+                      This is the default.
+
+  + `:master_slave`  :  Only the driver process, i.e. pid 1 connects to the
+                        workers. The workers do not connect to each other.
+
+  + `:custom`  :  The `launch` method of the cluster manager specifes the
+                  connection topology via fields `ident` and `connect_idents` in
+                  `WorkerConfig`. A worker with a cluster manager identity `ident`
+                  will connect to all workers specified in `connect_idents`.
```

Comment:
Would like to have nested lists here. The markdown syntax for the same is not yet supported?